components/sellercloud/sources/inventory-updated/inventory-updated.mjs

```diff
@@ -0,0 +1,71 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "sellercloud-inventory-updated",
+  name: "New Inventory Updated",
```

Comment:
```suggestion
  name: "Inventory Updated",
```
I think we can omit the "new" keyword here, since it's not actually a new item but rather an update.

I know our ESLint plugin warns source names should start with "New", but I understand it's a guideline, not a rule, and some names (such as this one) are more straightforward without it. What do you think?